src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -140,4 +146,41 @@ public void getProjectName_ReturnsProjectName() {
         // then
         assertThat(name).isEqualTo(projectName);
     }
+    
+    @Test
+	public void getTagsToExcludeFromChart_defaultConfiguration_ReturnsEmptyList() throws Exception {
+        // give
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+
+        // when
+        Collection<Pattern> patterns = configuration.getTagsToExcludeFromChart();
+
+        // then
+        assertThat(patterns).isEmpty();
+	}
+    
+    @Test
+	public void getTagsToExcludeFromChart_addPatterns_ReturnsListWithAllPatterns() throws Exception {
+        // give
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+        String somePattern = "@specificTagNameToExclude";
+        String anotherPattern = "@some.Regex.Pattern";
+        configuration.setTagsToExcludeFromChart(somePattern, anotherPattern);
+        
+        // when
+        Collection<Pattern> patterns = configuration.getTagsToExcludeFromChart();
+
+        // then
+        assertThat(patterns).extractingResultOf("pattern").containsOnly(somePattern, anotherPattern);
+	}
+    
+    @Test
+	public void setTagsToExcludeFromChart_invalidRegexPattern_ThrowsValidationException() throws Exception {
```

Comment:
1. will do
2. throwing ValidationException that wraps the PatternSyntaxException